packages/cli/src/commands/pg/settings/auto-explain/log-triggers.ts

```diff
@@ -11,8 +11,8 @@ export default class LogTriggers extends PGSettingsCommand {
   `)
 
   static args = {
-    database: Args.string(),
-    value: Args.string(),
+    database: Args.string({description: 'The config var exposed to the owning app containing the database configuration.'}),
+    value: Args.string({description: 'Boolean indicating whether trigger execution statistics should be enabled.'}),
```

Comment:
Change to "boolean indicating if the database has trigger execution statistics enabled"